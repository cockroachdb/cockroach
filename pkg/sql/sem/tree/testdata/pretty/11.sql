select fn(1, 2)
