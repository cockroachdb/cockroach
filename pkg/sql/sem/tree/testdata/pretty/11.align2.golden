1:
-
SELECT
	fn(
		1,
		2
	)

10:
----------
SELECT fn(
		1,
		2
       )

15:
---------------
SELECT fn(1, 2)
